Source/Tooling/Plugin/turnOffLogging.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import {turnOffLogging} from '@dolittle/tooling.common.logging'
+
+turnOffLogging();
```

Comment:
What does this thing do? From the looks of it, it would turn of logging for all other plugins as well?

Reply:
It's kinda a hack we need to do in plugins unless you want lots of logging everywhere. It will not turn of logging for other plugins nor the "host" of the running tooling application (e.g cli or vscode extension). Reason is that all the plugins have their own tooling platform dependencies, not sharing the host's

Reply:
That's just have node dependencies work

Reply:
IMO I think that they should share the dependency somehow, but it's not entirely clear to me how we can achieve that